docs/using/administering/upgrading.rst

```diff
@@ -0,0 +1,34 @@
+=========
+Upgrading
+=========
+
+A Flocker cluster can be upgraded to a newer version of Flocker while preserving all data and configuration of the Flocker cluster provided certain steps are followed.
+The correct steps to follow may vary depending on the versions of Flocker being upgraded from and to.
+A common requirement for all currently supported upgrade paths is that all nodes and the control service must be running the same version of Flocker.
```

Comment:
This sounds like an important piece of information related to upgrading. Please move this down a line so it starts a new paragraph.